scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1053,31 +1390,32 @@ function Get-AgentRegistry {
     return 'index.docker.io'
 }
 
-function Remove-IotEdgeContainers {
-    $dockerExe = Get-DockerExePath
-
-    if (-not (Test-IsDockerRunning 6> $null)) {
-        return
-    }
-
+function Stop-EdgeContainer([string] $Name = $null) {
```

Comment:
Change `$null` to `''`. (I know it doesn't make a difference but it helps avoid https://github.com/Azure/iotedge/pull/807/commits/f5ee113e46975733b2106acb8f29792789428e4a :) )

Reply:
Fixed.